models/multitask_centernet/multitask_centernet.py

```diff
@@ -0,0 +1,207 @@
+        xv, yv = np.meshgrid(np.arange(ny), np.arange(nx))
+        return np.stack((xv, yv), 2).reshape((-1, 2)).astype(np.float32)
+
+    def preprocess(self, img):
+        img = cv2.cvtColor(img, cv2.COLOR_BGR2RGB)
+        img = img.astype(np.float32) / 255.0
+        return img
+
+    def postprocess(self, frame, outs, padsize=None):
+        frameHeight = frame.shape[0]
+        frameWidth = frame.shape[1]
+        newh, neww, padh, padw = padsize
+        ratioh, ratiow = frameHeight / newh, frameWidth / neww
+        # Scan through all the bounding boxes output from the network and keep only the
+        # ones with high confidence scores. Assign the box's class label as the class with the highest score.
+
+        person_confidences, kp_confidences = [], []
+        person_boxes, kp_boxes = [], []
+        person_classIds, kp_classIds = [], []
+        person_rowinds = []
+        for i in range(outs.shape[0]):
+            detection = outs[i, :]
+            scores = detection[5:self.last_ind]
+            classId = np.argmax(scores)
+            confidence = scores[classId] * detection[4]
+            if classId == 0:
+                if detection[4] > config['person_conf_thres'] and confidence > config['person_conf_thres']:
+                    center_x = int((detection[0] - padw) * ratiow)
+                    center_y = int((detection[1] - padh) * ratioh)
+                    width = int(detection[2] * ratiow)
+                    height = int(detection[3] * ratioh)
+                    left = int(center_x - width * 0.5)
+                    top = int(center_y - height * 0.5)
+
+                    person_confidences.append(float(confidence))
+                    person_boxes.append([left, top, width, height])
+                    person_classIds.append(classId)
+                    person_rowinds.append(i)
+            else:
+                if detection[4] > config['kp_conf_thres'] and confidence > config['kp_conf_thres']:
+                    center_x = int((detection[0] - padw) * ratiow)
+                    center_y = int((detection[1] - padh) * ratioh)
+                    width = int(detection[2] * ratiow)
+                    height = int(detection[3] * ratioh)
+                    left = int(center_x - width * 0.5)
+                    top = int(center_y - height * 0.5)
+
+                    kp_confidences.append(float(confidence))
+                    kp_boxes.append([left, top, width, height])
+                    kp_classIds.append(classId)
+
+        # Perform non maximum suppression to eliminate redundant overlapping boxes with
+        # lower confidences.
+        person_indices = cv2.dnn.NMSBoxes(person_boxes, person_confidences, config['person_conf_thres'],
+                                          config['person_iou_thres']).flatten()
+        kp_indices = cv2.dnn.NMSBoxes(kp_boxes, kp_confidences, config['kp_conf_thres'],
+                                      config['kp_iou_thres']).flatten()

```

Comment:
`NMSBoxes` returns a empty tuple if no person is detected. Calling `flatten` on an empty tuple triggers error.
Traceback (most recent call last):
```
  File "/path/opencv_zoo/models/multitask_centernet/demo.py", line 14, in <module>
    srcimg = mcn.detect(srcimg)
  File "/path/opencv_zoo/models/multitask_centernet/multitask_centernet.py", line 207, in detect
    srcimg = self.postprocess(srcimg, outs, padsize=(newh, neww, padh, padw))
  File "/path/opencv_zoo/models/multitask_centernet/multitask_centernet.py", line 112, in postprocess
    kp_indices = cv2.dnn.NMSBoxes(kp_boxes, kp_confidences, config['kp_conf_thres'],
AttributeError: 'tuple' object has no attribute 'flatten'
```

Please doublecheck with images with no person and even no objects at all.